lib/src/layer/tile_layer.dart

```diff
@@ -952,11 +1034,14 @@ class Tile implements Comparable<Tile> {
     _imageStream?.removeListener(_listener);
```

Comment:
I think we also need to dispose of the animation controller, that was a reason for crash